cmd2/parsing.py

```diff
@@ -105,6 +105,17 @@ def command_and_args(self):
             rtn = None
         return rtn
 
+    @property
+    def arg_list(self) -> Union[List[str], None]:
```

Comment:
I think this should always return a list, possibly an empty one but never None.  Doing so would require extra checks in things that call it such as in do-shell now.

Reply:
Done